code/modules/cm_aliens/structures/special/spawn_pool.dm

```diff
@@ -185,4 +187,41 @@
 		visible_message(SPAN_XENODANGER("You hear something resembling a scream from [src] as it's destroyed!"))
 		xeno_message(SPAN_XENOANNOUNCE("Psychic pain storms throughout the hive as the spawn pool is destroyed! You will no longer gain pooled larva over time."), 3, linked_hive.hivenumber)
 		linked_hive.hijack_pooled_surge = FALSE
+
+	for(var/turf/T in range(1, src))
+		var/obj/effect/xenomorph/spray/spray = new /obj/effect/xenomorph/spray(T)
+		spray.cause_data = create_cause_data(src)
```

Comment:
This is a `Destroy` proc, you can't pass `src` outside of it, it's against the point

I think `create_cause_data` takes a name as first argument, maybe you meant `src.name` or `"[src]"` ?